tsl/src/README.module.md

```diff
@@ -0,0 +1,34 @@
+# Submodule Licensing and Initialization #
+
+## Loading and Activation ##
+
+We link module loading and activation to the license GUC itself.
+While keeping multiple GUCs, e.g., one for module loading and one licensing,
+is tempting, it leads to hairy edge cases when the GUCs disagree.
```

Comment:
Is it necessary to have this sentence discussing what is not done? May be just focus what is available?